src/app/component/eco-news/create-news/create-news-cancel/create-news-cancel.component.ts

```diff
@@ -1,6 +1,7 @@
 import { Component, OnInit } from '@angular/core';
```

Comment:
Bug #686 isn't fixed, please check it one more time

Reply:
Also, please, select 3 tags. After that deselect 2 tags and you'll see that 'Publish' button is disabled. But one tag is still selected. It should be also fixed.

Reply:
fixed